hogben/contrasts.py

```diff
@@ -142,19 +142,19 @@ def _figure_2(save_path):
                          for angle, points, split in angle_splits]
 
     # Information from the D2O and H2O contrasts for each model.
-    g_init_1 = (sample_1.angle_info(d2o_angle_times_1, [6.36]) +
-                sample_1.angle_info(h2o_angle_times_1, [-0.56]))
+    g_init_1 = (sample_1.angle_info(d2o_angle_times_1, [6.36]).g +
+                sample_1.angle_info(h2o_angle_times_1, [-0.56]).g)
 
-    g_init_2 = (sample_2.angle_info(d2o_angle_times_2, [6.36]) +
-                sample_2.angle_info(h2o_angle_times_2, [-0.56]))
+    g_init_2 = (sample_2.angle_info(d2o_angle_times_2, [6.36]).g +
```

Comment:
I was about to leave a comment on this, but then realised that we also have an issue to remove this whole function